rancher2/schema_cluster_v2_rke_config.go

```diff
@@ -122,6 +122,15 @@ func clusterV2RKEConfigFields() map[string]*schema.Schema {
 				Schema: clusterV2RKEConfigETCDFields(),
 			},
 		},
+		"rotate_certificates": {
```

Comment:
Does `rotate_certificates` have a default bool value in rancher?

Reply:
No the object is nilable.